### PR TITLE
Feature/cra 3

### PR DESCRIPTION
**Pull request by MaciekPon**
I have added section to main page with title, image and some description

### CODE REVIEW

File: src/modules/main/components/EncourageSection/EncourageSection.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+import styles from "./EncourageSection.module.scss";
```
Comment (ruljin) [inline src/modules/main/components/EncourageSection/EncourageSection.jsx]: I would recommend changing the CSS file name to `section.module.scss` and this file to `Section` or if it is a specific use-case component than to `enourage_section.module.scss` and the name of the component can stay then.

File: src/modules/main/components/EncourageSection/EncourageSection.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export const EncourageSection = ({ image, title, description }) => {
```
Comment (ruljin) [inline src/modules/main/components/EncourageSection/EncourageSection.jsx]: You don't pass `image` but `imageSrc` and the alt attribute can be just empty.

**Review by ruljin — CHANGES_REQUESTED** [review]
Few things to change

File: src/modules/main/components/EncourageSection/EncourageSection.jsx (hunk 1 above)
Comment (MaciekPon) [reply src/modules/main/components/EncourageSection/EncourageSection.jsx]: I agree with You that `Section.jsx `and `section.module.scss` should be enough

**Review by ruljin — APPROVED** [review]

**Pull request merged**